ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/conversion_filters/109473_filter-by-delivery-officer.cy.js

```diff
@@ -0,0 +1,35 @@
+/// <reference types ='Cypress'/>
+
+Cypress._.each(['ipad-2'], (viewport) => {
+    describe(`109473: Filter projects by deliver officer ${viewport}`, () => {
+      beforeEach(() => {
+        cy.viewport(viewport);
+        cy.login();
+        cy.navigateToFilterProjects();
+      })
+  
+      afterEach(() => {
+        cy.clearCookies();
+      });
+  
+      it('TC01: should display Not assigned filter option at the top', () => {
+        cy.get(':nth-child(3) > .govuk-form-group > .govuk-fieldset > .govuk-checkboxes > :nth-child(1)')
```

Comment:
Suggest using '[for=filter-delivery-officer-not-assigned]' as a selector.
It would be good to avoid complicated selectors if we can, they tend to break easily if the structure of the html changes later down the line.

Reply:
I agree however here as a tester I want to check the very first checkbox text and using this selector doesn't solve my purpose as it's hard coded. I think I can try getting the selector at the fist position